dizoo/mujoco/config/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .ant_ddpg_config import ant_ddpg_config
+from .ant_ddpg_config import ant_ddpg_config, ant_ddpg_create_config
```

Comment:
can we remove contents in `__init__py` for mujoco? It is not be used in unittest so we don't need these import lines